problem1/problem1_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "./problem1"
+describe Problem1 do
+	it "Sum Even Numbers" do
```

Comment:
Take a quick look at [better specs](http://www.betterspecs.org/) and take a stab at improving your specs.